server/admin.api.js

```diff
@@ -138,6 +139,150 @@ router.get('/admin/stats', mw.adminOnly, async (req, res)=>{
 	}
 });
 
+router.get('/api/lock/count', mw.adminOnly, async (req, res)=>{
+	try {
+		const countLocksQuery = {
+			lock : { $exists: true }
+		};
+		const count = await HomebrewModel.countDocuments(countLocksQuery)
+			.then((result)=>{
+				return result;
+			});
+		return res.json({
+			count
+		});
+	} catch (error) {
+		console.error(error);
+		return res.json({ status: 'ERROR', detail: 'Unable to get lock count', error });
+	}
+});
```

Comment:
Might I suggest the following, and a similar pattern to the other try/catch blocks?

```suggestion
router.get('/api/lock/count', mw.adminOnly, async (req, res)=>{
	const countLocksQuery = { lock : { $exists: true } };
	const count = await HomebrewModel.countDocuments(countLocksQuery);
		.catch (error) {
			console.error(error);
			return res.json({ status: 'ERROR', detail: 'Unable to get lock count', error });
		}
	
	if (count)
		return res.json({count});
});
```

Reply:
This pattern is applied in the latest commits.